docs/diagrams/StorageClassDiagram.puml

```diff
@@ -18,7 +18,7 @@ StorageManager o--> AddressBookStorage
 
 JsonUserPrefsStorage .left.|> UserPrefsStorage
 JsonAddressBookStorage .left.|> AddressBookStorage
-JsonAddressBookStorage .down.> JsonSerializableAddressBookStorage
-JsonSerializableAddressBookStorage .right.> JsonSerializablePerson
-JsonSerializablePerson .right.> JsonAdaptedTag
+JsonAddressBookStorage .down.> JsonSerializableAddressBook
+JsonSerializableAddressBook .right.> JsonAdaptedCustomer
+JsonAdaptedCustomer .right.> JsonAdaptedTag
```

Comment:
https://github.com/nus-cs2103-AY2021S2/forum/issues/273

As per must-see issues in tp forum

Incorrect class name in storage class diagram